src/api/index.ts

```diff
@@ -33,6 +33,9 @@ export const getDefaultProvider = (): string | null =>
 function createWeb3Api(): Web3 {
   // TODO: Create an `EthereumApi` https://github.com/gnosis/dex-react/issues/331
   const web3 = new Web3(getDefaultProvider())
+  // `handleRevert = true` makes `require` failures to throw
+  // For more details see https://github.com/gnosis/dex-react/issues/511
+  web3.eth['handleRevert'] = true
```

Comment:
Is it a new addition, seeing as it's not even typed?

Reply:
never seen this

Reply:
Not sure, it's on the docs though https://web3js.readthedocs.io/en/v1.2.5/web3-eth.html#handlerevert